icubam/backoffice/handlers/base.py

```diff
@@ -21,3 +24,15 @@ def get_user_locale(self):
     locale = self.get_query_argument('hl', default=self.config.locale)
     # We fallback to Accept-Language header.
     return tornado.locale.get(locale) if locale else None
+
+  def set_default_headers(self):
```

Comment:
nice, btw, after we should try to figure out how to have it glued to icubam website, still running another process..

Reply:
do you want a single server?
Fine for me, I will have to check the structure of the website. Let's open issue/ or something to track that.